addons/account/models/ir_actions_report.py

```diff
@@ -47,7 +47,8 @@ def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
         return collected_streams
 
     def _is_invoice_report(self, report_ref):
-        return self._get_report(report_ref).is_invoice_report
+        report = self._get_report(report_ref)
+        return report.is_invoice_report or report.report_name == 'account.report_invoice_with_payments'
```

Comment:
```suggestion
        return report.is_invoice_report or report.report_name == 'account.report_invoice'
```
The name of the report is `'account.report_invoice'`